src/main/scala/com/ncredinburgh/sonar/scalastyle/ScalastyleRepository.scala

```diff
@@ -89,15 +89,16 @@ class ScalastyleRepository extends RulesDefinition {
   private def determineFreeRuleKey(clazz: String, repo: NewRepository): String = {
     @tailrec
     def getFreeRuleKey(key: String, count: Int, repo: NewRepository): String = {
-      val testee = if (count == 0) key else "$key_$count"
-      if (repo.rule(testee) == null) {
-        testee
+      val ruleKey = if (count == 0) key else s"${key}_${count}d"
```

Comment:
A redundant `d` at the end of the string?

Reply:
Also, how did it work before without the interpolation? 😮

Reply:
Oh yeah, Jesus my dyslexia will kill me some day.:sweat:
And I supose that you never need the interpolation before